server/controllers/student/addStudent.js

```diff
@@ -0,0 +1,24 @@
+const { insertStudent } = require('../../database/queries/addStudent');
+
+const addStudent = (req, res) => {
+  insertStudent({ ...req.body })
+    .then((student) => {
+      if (!student.rowCount) {
+        res.status(400).send({
+          error: 'Student dose not added',
```

Comment:
you should be more specific in error msg like you can check the error code and rely on it to determine the error. 
like : 
https://www.postgresql.org/docs/8.4/errcodes-appendix.html